v0.39lh

lheijst opened the pull request:
2016-05-31 v0.39lh

Changelog:
- Change: replaced calculate_soil_moisture by lookup_potential
- Change: replaced calculate_leaf_wetness by lookup_potential
- Change: no debug message if rain_count=128 (128=no sensor)
- Change: set wind_dir to None when wind_speed == 0 (or < 0.001)
- Added: set leaf_temp = soil_temp in machine format for channels 1 and 2
- Change: soil moisture potential calulated with norm factor 0.009
- Change: leaf wetness potential calulated with norm factor 0.0

lheijst committed:
version 0.39lh

lheijst committed:
version 0.39lh
No debug at rain_count==128

matthewwall commented:
luc,  the policy of windDir=None when windSpeed is zero is handled by StdWXCalculate.  it should not be in the driver code.

lheijst commented:
OK, thanks. Just copied this from the vantage driver. :-)

lheijst committed:
2016-06-09 v0.40lh

Changelog:

Added: routine new_archive_record
Changed: handling of rf_stats moved to new_archive_record
Changed: SM-MAP and LW_map settings
Changed: all debug levels from 1 to 2
Changed: removed rainRate from DEFAULT_SENSOR_MAP
Changed: packets with only a battery status are now skipped
Changed: made handling of rain_count more robust
Changed: pctgood is now rounded to the nearest integer value
Added: Raw rainrate value checked for no sensor
Fixed: battery status for leaf-soil station was missing
Changed: Removed debug code in leaf-soil section

matthewwall commented:
changes taken from separate meteostick.py submission, so this change request is now moot.
